src/LodeRunner/src/ValidationTest/Main.cs

```diff
@@ -474,6 +474,7 @@ public PerfLog CreatePerfLog(string server, Request request, ValidationResult va
                 Duration = duration,
                 ContentLength = contentLength,
                 Failed = validationResult.Failed,
+                LoadClientId = this.config.LoadClientId,
```

Comment:
run code cleanup to remove "this."

Reply:
Since I've been back from break, I've been adding `this` myself, because stylecop has started erroring for it and with the rule: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1101.md.  So, it might be there, because it got marked.

Reply:
@jofultz I think this is a case of having an out of date cop file.  When we reviewed the rules almost a year ago, we decided to not use this (thus my comment).  I'm OK either way as long as we're consistent. If you don't use this, you sometimes end up with a few for clarity which is OK.

Reply:
@jofultz I think VS code clean up will auto fix these if configured correctly

Reply:
https://github.com/retaildevcrews/wcnp/issues/556